pkg/app/run.go

```diff
@@ -113,6 +114,18 @@ func Run(opts RunOpts) error {
 		return nil
 	}
 
+	if !g.Config.CLI.MetadataNoticeDisplayed && commandCollectsData(commandName) {
+		// FIXME: We need the actual URL to point users to.
+		text.Important(g.Output, "The Fastly CLI is configured to collect data related to Wasm builds (e.g. compilation times, resource usage, and other non-identifying data). To learn more about what data is being collected, why, and how to disable it: https://www.fastly.com/")
```

Comment:
@jaskiratr we'll need an actual URL here (or should this be https://bit.ly/wasm-metadata which points to our community forum thread on the topic)?

Reply:
We'll update the [CLI reference page](https://developer.fastly.com/reference/cli/) with the information about what data is being collected, why, and how to disable it